lib/HLSL/DxilSignature.cpp

```diff
@@ -43,6 +43,19 @@ unique_ptr<DxilSignatureElement> DxilSignature::CreateElement() {
   return unique_ptr<DxilSignatureElement>(new DxilSignatureElement(m_sigPointKind));
 }
 
+void DxilSignature::CopySignatureElements(DxilSignature &src) {
+  const bool bSetID = false;
+  for (auto &Elt : src.GetElements()) {
+    std::unique_ptr<DxilSignatureElement> newElt = CreateElement();
```

Comment:
Should we add a copy constructor for `DxilSignatureElement` to simplify this code?